IdentityCore/src/oauth2/token/MSIDAccessTokenWithAuthScheme.m

```diff
@@ -105,8 +105,8 @@ - (instancetype)initWithTokenCacheItem:(MSIDCredentialCacheItem *)tokenCacheItem
             return nil;
         }
         
-        _accessTokenType = tokenCacheItem.accessTokenType;
-        if (!_accessTokenType)
+        _tokenType = tokenCacheItem.tokenType;
+        if (!_tokenType)
```

Comment:
Should token type be optional?

Reply:
I think token type is essential because it is now saved as part of the cache value for pop tokens . token_type = @"Pop".